src/utils/asyncTimer.js

```diff
@@ -1,6 +1,6 @@
 
-const asyncTimer = (timeout) => new Promise(
-  (resolve) => setTimeout(() => resolve(), timeout),
+const asyncTimer = (timeout, body = 'OK') => new Promise(
```

Comment:
why do you add this ok? just for rebuilding?

Reply:
No, I don't have the permission to delete builds in ECR. We had this problem several times before, and I've asked Marcell how to solve this issue. This is the recommended way - merge in a dummy PR. But because we now have codecov, we can't just merge in any PR.

Reply:
I add the 'OK' primarily to help with testing, so that I can check that the function resolves to the vale that's passed to it. This function right now is used only once in the API and its return value is not used anywhere.